stablepay-sdk/src/styles/PricingCard.css

```
@@ -1,361 +1,475 @@
+  width: 48px;
+  height: 48px;
   background-image: url("../assets/stablepay-logo.svg");
   background-position: center;
   background-size: contain;
   background-repeat: no-repeat;
+  border-radius: 12px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.05);
 }
 
 .stablepayTitle {
   font-family: "Inter", sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)